pkg/sm/sm.go

```diff
@@ -191,6 +197,17 @@ func (smb *ServiceManagerBuilder) Build() *ServiceManager {
 	srv := server.New(smb.cfg.Server, smb.API)
 	srv.Use(filters.NewRecoveryMiddleware())
 
+	// start the operation maintainer
+	smb.OperationMaintainer.Run()
+
+	// start each Controller scheduler's worker pool
+	for _, controller := range smb.API.Controllers {
```

Comment:
Provide different configurations for each scheduler's pool when creating it. It will allow for fine grained configuration per resource

Reply:
I'll have to think about a **clear** way to provide such a configuration.